VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -654,4 +654,10 @@
   <data name="TestMethodWithoutTestAttributeAnalyzerTitle" xml:space="preserve">
     <value>A method might be missing a test attribute.</value>
   </data>
+  <data name="GetHashCodeRefersToMutableFieldAnalyzerMessage" xml:space="preserve">
+    <value>GetHashCode() refers to {0}</value>
```

Comment:
It should still be a somewhat different message: "GetHashcode() refers to <blub>" doesn't make sense from an error message perspective. Even "GetHashcode() refers to the discouraged field/prop <blub>" is better.